manifests/golang.yml

```
@@ -406,20 +418,20 @@ tests/:
       Test_UserEventsStandalone_SDK_V1: missing_feature
       Test_UserEventsStandalone_SDK_V2: missing_feature
     test_automated_login_events.py:
-      Test_Login_Events: missing_feature
-      Test_Login_Events_Extended: missing_feature
-      Test_V2_Login_Events: missing_feature
-      Test_V2_Login_Events_Anon: missing_feature
-      Test_V2_Login_Events_RC: missing_feature
-      Test_V3_Auto_User_Instrum_Mode_Capability: missing_feature
-      Test_V3_Login_Events: missing_feature
-      Test_V3_Login_Events_Anon: missing_feature
-      Test_V3_Login_Events_Blocking: missing_feature
-      Test_V3_Login_Events_RC: missing_feature
+      Test_Login_Events: irrelevant (Go only supports SDK events)
```

This is not true for the net-http-orchestrion variant

```suggestion
      Test_Login_Events:
        '*': irrelevant (Go only supports SDK events)
        net-http-orchestrion: missing_feature
```

---

I guess you're right.